tests/conftest.py

```diff
@@ -0,0 +1,128 @@
+# std
+import os
+from random import randint
+from pathlib import Path
+from json import load, dump, loads
+
+# vendor
+from moto import mock_dynamodb2
+import pytest
+from pytest import fixture
+import boto3
+from starlette.testclient import TestClient
+
+# local
+from app._version import __version__
+from app.schema import Game
+from app.utils import UniversalEncoder
+
+@fixture(autouse=True)
+def _aws_credentials(monkeypatch):
+    """Mocked AWS Credentials for moto."""
+    monkeypatch.setitem(os.environ, "AWS_ACCESS_KEY_ID", "testing")
+    monkeypatch.setitem(os.environ, "AWS_SECRET_ACCESS_KEY", "testing")
+    monkeypatch.setitem(os.environ, "AWS_SECURITY_TOKEN", "testing")
+    monkeypatch.setitem(os.environ, "AWS_SESSION_TOKEN", "testing")
+    yield
+
+
+
+@fixture
+def create_table():
+    # create dynamodb interface (mocked by moto decorator)
+    dynamodb = boto3.resource('dynamodb', 'us-east-1')
+    table_name = "GamesTest"
+
+    table = dynamodb.Table(table_name)
+    try:
+        if table.item_count >= 0:
+            print("table exists, deleting")
```

Comment:
A gentle encouragement to remove print statements from shared code. :) Preferred alternatives: use logging, raise an exception.

Reply:
This links back to a question I asked on the last PR (you may not have seen it since I asked it after I merged that one) about db setup and testing in general. You answered it partially below when discussing `test_get_no_game` -- I'll switch to using the full db in test. I think I'll still have some questions in this vein though, and I'll post them here tomorrow

Reply:
Cool, yeah, I never quite grok'ed how dbsetup was being used.

Reply:
OK, so I think my remaining question boils down to: should I include in this repo a script for initial (non-test) database setup? Sort of an "install script"? Back in my PHP days this was something people did.

Reply:
Yeah – figuring out how to manage the database schema alongside your code changes is always a thing. If you were using SQL, my suggestion would be to look into a framework like Alembic, which is designed to help you manage (initialize and migrate) database changes. If you were building a larger project where you needed to manage a lot of infrastructure, I'd encourage you to look at Terraform for keeping your cloud infrastructure (including DynamoDB resources) up to date.

So I guess my question now is: what do you want a developer's workflow to look like? If you think they can be effective in making changes using the mock infrastructure that accompanies your unit tests, maybe that's all you need. If you think they need to run a server locally without pointing it to a hosted database (you could *also* point developers to a cloud-hosted staging area), then maybe you don't need to include management scripts in your database.

If you *do* decide to include management scripts like the one above, I would encourage you to look into extending Quart's custom command line tree: https://pgjones.gitlab.io/quart/command_line.html (which uses a python package called 'click' under the covers).

Reply:
for now, no db management scripts, tho i may come back to this